lib/Backend/GlobOpt.cpp

```diff
@@ -5166,9 +5174,15 @@ GlobOpt::OptInstr(IR::Instr *&instr, bool* isInstrRemoved)
 
     if (instr->HasBailOutInfo() && !this->IsLoopPrePass())
     {
-        this->currentBlock->globOptData.changedSyms->ClearAll();
-        this->currentBlock->globOptData.capturedValues = 
-            this->currentBlock->globOptData.capturedValuesCandidate;
+        GlobOptBlockData * globOptData = &this->currentBlock->globOptData;
+        globOptData->changedSyms->ClearAll();
+        globOptData->changedSyms->Or(this->changedSymsAfterIncBailoutCandidate);
```

Comment:
Remove Or() and use swap instead to reduce the cost.

---
In reply to: [72336744](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72336744) [](ancestors = 72336744)